### ISSUE TITLE
US Active cases are 0s

### ISSUE THREAD

**Issue by galalatef** [CSSEGISandData/COVID-19#1658]
US active cases are being zeros

**Comment by harsh306** [comment]
++1, and also the state-level information is missing recently. Can any take a look at this?

**Comment by RoStall** [comment]
At least part of this is covered here, https://github.com/CSSEGISandData/COVID-19/issues/1250

Unknown as to when said changes are to be implemented.

Kudos to the repo owners for taking on the task

**Comment by harsh306** [comment]
I agree, I really appreciate all the people who are maintaining this dataset. I also found this link: https://github.com/coviddata/covid-api ,  again by John Hopkins. This should be helpful to the people facing this issue.
